R/vendor.R

```diff
@@ -48,16 +51,37 @@ cpp_vendor <- function(path = ".") {
 
   cpp11_header <- sprintf("// cpp11 version: %s\n// vendored on: %s", cpp11_version, Sys.Date())
 
-  files <- list.files(current, full.names = TRUE)
+  write_header(path, "cpp11.hpp", "cpp11", cpp11_header)
+
+  copy_files(list.files(current, full.names = TRUE), path, "cpp11", cpp11_header)
+
+  # Additional steps to make vendoring work ----
+
+  message(paste(
+    "Makevars and/or Makevars.win should have a line such as",
+    "'PKG_CPPFLAGS = -I../inst/include'"
```

Comment:
Does this now need to show the new path?

Reply:
that line is added automatically, and uses the path the user wants

Reply:
i reallize i never pushed that change, done now